lapce-app/src/palette/kind.rs

```diff
@@ -19,6 +24,7 @@ impl PaletteKind {
     /// The symbol/prefix that is used to signify the behavior of the palette.
     pub fn symbol(&self) -> &'static str {
         match &self {
+            PaletteKind::PaletteHelp => "?",
```

Comment:
`?` is GlobalSearch prefix

Reply:
What prefix would you like to be used?

Reply:
It's not currently used because it's commented
![image](https://github.com/lapce/lapce/assets/50457605/7cab8f63-1321-4e36-a541-859c092cfdd1)

Reply:
> What prefix would you like to be used?

I discussed with @panekj and `?` is fine.